### PR TITLE
[Kotlin] Add `annalyns-infiltration` concept exercise

### PR DESCRIPTION
**Pull request by dector**
Shamelessly ~~stolen~~ copied from C# version. 😄

### CODE REVIEW

**Comment by dector** [comment]
@ErikSchierboom I guess only you can save me here 😄

File: languages/kotlin/exercises/concept/annalyns-infiltration/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "authors": [
+    {
+      "github_username": "ErikSchierboom",
+      "exercism_username": "ErikSchierboom"
```
Comment (ErikSchierboom) [inline languages/kotlin/exercises/concept/annalyns-infiltration/.meta/config.json:5]: You should put your own name here, as _you_ are the author of this exercise (i.e. you've done the work to create it). I will be credited indirectly through the `forked_from` exercise, of which I'm the author.

**Review by ErikSchierboom — APPROVED** [review]
Awesome work!

File: languages/kotlin/exercises/concept/annalyns-infiltration/.meta/config.json (hunk 1 above)
Comment (dector) [reply languages/kotlin/exercises/concept/annalyns-infiltration/.meta/config.json:5]: Got it. Will change

**Pull request merged**